docs/protocol/off-chain/channel-methods.mdx

```diff
@@ -0,0 +1,681 @@
+
+**Example**:
+```json
+{
+  "participants": ["0x742d35Cc...", "0x123456Cc..."],
+  "adjudicator": "0xAdjudicator123...",
+  "challenge": 86400,
+  "nonce": 1699123456
+}
+```
+
+#### State {#state-structure}
+
+| Field | Type | Description | Notes |
+|-------|------|-------------|-------|
+| `data` | bytes | Contains CHANOPEN magic number (7877 / 0x1EC5) | Signals this is a funding state |
+| `allocations` | StateAllocation[] | Expected deposits from each participant | Order matches participants array<br/>See [StateAllocation](#stateallocation) below<br/>Clearnode typically deposits 0 initially |
+| `sigs` | Signature[] | Array of signatures; the clearnode's at index 1 | Order: Index matches participants array<br/>User must add their signature at index 0 before submitting on-chain |
+
+**Example**:
+```json
+{
+  "data": "0x0000000000000000000000000000000000000000000000000000000000001ec5",
+  "allocations": [
+    {"destination": "0x742d35Cc...", "token": "0x2791Bca1...", "amount": "100000000"},
+    {"destination": "0x123456Cc...", "token": "0x2791Bca1...", "amount": "0"}
+  ],
+  "sigs": [null, "0xClearnodeSignature123..."]
+}
+```
+
+#### StateAllocation {#stateallocation}
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `destination` | wallet address | Participant's wallet address |
+| `token` | wallet address | Token contract address |
+| `amount` | string | Amount in smallest unit (e.g., `"100000000"` for 100 USDC with 6 decimals) |
+
+:::tip Clearnode Signature First
+<Tooltip content="A virtual ledger layer that provides a unified ledger (through Nitro RPC) and coordinates state channels (through Nitrolite), providing chain abstraction for developers and users">The clearnode</Tooltip> provides its signature BEFORE the user commits funds on-chain. This ensures both parties have committed to the channel before any on-chain transaction occurs.
+:::
+
+### Next Steps After Receiving Response
+
+1. **Verify Channel Data**
+   - Recompute `channelId` = `keccak256(abi.encode(channel))`
+   - Verify computed ID matches response `channel_id`
+   - Check participants[0] is your wallet address
+   - Verify token and amount match your request
+
+2. **Verify the Clearnode's Signature**
+   - Compute `stateHash` = `keccak256(abi.encode(channelId, state.intent, state.version, state.data, state.allocations))`
+   - Recover signer from `broker_signature`
+   - Verify signer is <Tooltip content="A virtual ledger layer that provides a unified ledger (through Nitro RPC) and coordinates state channels (through Nitrolite), providing chain abstraction for developers and users">the clearnode's</Tooltip> known wallet address
+
+3. **Sign State with Your Key**
+   - Sign `stateHash` with your <Tooltip content="An entity (identified by a wallet address) that is part of a channel">participant</Tooltip> key
+   - Add your signature at `state.sigs[0]`
+
+4. **Approve Token Spending** (if needed)
```

Comment:
It cannot be needed on this stage in the current version of the protocol as this action won't do any funds movement